docs/content/docs/getting-started/use-built-provider.md

```diff
@@ -54,6 +54,43 @@ ln -s $GOPATH/bin/terraform-provider-google-beta ~/.terraform.d/plugins/registry
 
 Once this setup is complete, terraform will automatically use the binaries generated by the `make build` commands in the `terraform-provider-google` and `terraform-provider-google-beta` repositories instead of downloading the latest versions. 
 
+### Alternative setup
```

Comment:
This page has changed substantially recently, and this section may be obviated: https://googlecloudplatform.github.io/magic-modules/docs/getting-started/use-built-provider/

Reply:
Yea, I think this section is now covered in the latest version of this same file https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/use-built-provider.md. @rosmo unless you think there are specific pieces you'd like to weave in to the new version, we can probably remove it from this PR.